platform/process/tasks/ABProcessTaskEmail.js

```diff
@@ -265,4 +278,27 @@ module.exports = class ABProcessTaskEmail extends ABProcessTaskEmailCore {
             });
       });
    }
+
+   processMessageText(instance, message) {
+      const previousElement = this.process
+         .elements()
+         .filter(
+            (e) =>
+               e instanceof ABProcessTaskServiceGetResetPasswordUrl ||
+               e instanceof ABProcessTaskServiceQuery
+         );
+      const previousProcessData = {};
+
+      for (let i = 0; i < previousElement.length; i++) {
+         const key = previousElement[i].name.replaceAll(" ", "_");
```

Comment:
Is ( ) space always once in the text ? 

Should we use `.replaceAll(\ \g, "_");` ?